app/design/frontend/Clothes/touch/Magento_Cms/templates/default/mag15.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+?>
+
+
+
+<div class="mag15<?= $block->getData('argumentClassFirst') ?><?= $block->getData('argumentClassSecond') ?>"><p>MAG-15 test. <?= $block->getData('argumentContentSecond') ?><?= $block->getData('argumentContentFirst') ?></p></div>
```

Comment:
@yuratouch This code also can be improved. Please, use 2 $block->getData() instead of 4

Reply:
Thanx for the advice! Now I get it that this way really better (less code) and code working as well.